src/geom/elem.C

```diff
@@ -130,7 +130,7 @@ const unsigned int Elem::type_to_n_nodes_map [] =
     18, // INFHEX18
 
     6,  // INFPRISM6
-    16, // INFPRISM12
+    12, // INFPRISM12
```

Comment:
Wow... not sure how that happened. I will backport this to the 1.6.2 release as well.

Reply:
Broken ever since Ben committed it in 2007, according to 2b80e9d296dca053a6661ab721db15850a28c7ff

Not sure how it survived this long, though.  I guess we don't have any Xdr/Checkpoint/DistributedMesh test coverage of InfPrism12?

Reply:
I don't think there are more tests than the `inf_fe_radial_tests` for inifinite elements? 
By the way: Would you expect that checkpointing works for infinite elements? I once tried, but there were several errors (that above coming first); so I gave up thinking that this just isn't implemented...

Reply:
I would not expect that.  I don't know of anything specifically precluding checkpointing working for infinite elements, so hopefully there aren't too many bugs to fix, but in my experience if something doesn't have complete test coverage there's never zero bugs.  The closest I've ever seen was the vector-valued FE periodic boundary code that we just turned on; it was utterly astonishing to see that that completely untested instantiation was actually correct and only needed to be enabled..